packages/components/src/components/input-field/input-field.jsx

```diff
@@ -132,7 +132,7 @@ const InputField = ({
         if (max_is_disabled) return;
         let increment_value;
 
-        const current_value = +(local_value || value);
+        const current_value = local_value || value;
```

Comment:
This change was made due to how number was rounded before. For e.g., if we pass '4.00' (type string) it will be just 4 (type number), without any numbers after decimal point. That behaviour changed the step of incrementing/ decrementing barrier value: 3.98 -> 3.99 -> 4 -> 5 ->... 
Now it's working properly: 3.98 -> 3.99 -> 4.01 -> 4.02 -> ...